util/container/Dockerfile

```diff
@@ -78,7 +61,7 @@ RUN apt-get update && \
         libtool \
         minicom \
         screen && \
-    apt-get clean ; \
+    apt-get clean && \
```

Comment:
That is actually intentional (not that it matters that much)

Reply:
Oh, good point. I'll put it back. Also (reading the Dockerfiles best practices page), I think there's a missing `apt-get update` which I'll add in a sec.